src/config/set.js

```diff
@@ -10,7 +10,14 @@ const CONFIGS = new Map([
     'defaultProvider',
     value => {
       const providers = require('../providers')
-      return providers.hasOwnProperty(value)
+      return [providers.hasOwnProperty(value), value]
+    }
+  ],
+  [
+    'includeProtocol',
```

Comment:
The first part of a URI that has the `https:`, `http:`, `file:`, `git:`, etc. is called a "scheme" by the way ;) Might be a bit more accurate here.

Reply:
Oh wow you're totally right. Wonder how long I've been saying that wrong 😬

Reply:
Haha no worries, I think it's referred to as a "protocol" in the context of URLs pretty frequently. But in this case, we'd like to use "scheme" :)